README.md

```diff
@@ -1 +1,3 @@
 # hooks-test3
+
+#### nigel wuz here
```

Comment:
I think this might be a typo? Did you mean `fuzz`?

Reply:
nigel don't make mistakes